paddle/fluid/inference/tests/api/analyzer_int8_image_classification_tester.cc

```diff
@@ -30,8 +28,13 @@ void SetConfig(AnalysisConfig *cfg) {
   cfg->SwitchIrOptim();
   cfg->SwitchSpecifyInputNames(false);
   cfg->SetCpuMathLibraryNumThreads(FLAGS_paddle_num_threads);
-
   cfg->EnableMKLDNN();
+  cfg->pass_builder()->SetPasses(
+      {"infer_clean_graph_pass", "mkldnn_placement_pass",
+       "depthwise_conv_mkldnn_pass", "conv_bn_fuse_pass",
+       "conv_eltwiseadd_bn_fuse_pass", "conv_bias_mkldnn_fuse_pass",
+       "conv_elementwise_add_mkldnn_fuse_pass", "conv_relu_mkldnn_fuse_pass",
+       "fc_fuse_pass", "is_test_pass"});
```

Comment:
Fix SetPass will in another PR?

Reply:
yes

Reply:
@luotao1 , it turned out that no special fix is required for this and the call to `SetPasses()` can be simply removed now. Some other modifications must have fixed the accuracy problem. Of course, repeated passes problem would still benefit from a cleanup, but it is not critical here.

Reply:
> We will prepare a PR with a cleanup later.

Got it. Does new PR fix https://github.com/PaddlePaddle/Paddle/pull/16559#discussion_r270747088 as well?

Reply:
I will fix the fps calculation and remove redundant `PredictionRun` method in this PR.
WIP

Reply:
Done.